saleor/plugins/manager.py

```diff
@@ -292,15 +293,32 @@ def calculate_checkout_shipping(
             checkout_info.checkout.currency,
         )
 
+    def calculate_order_total(
+        self,
+        order: "Order",
+        lines: Iterable["OrderLine"],
+    ) -> TaxedMoney:
+        currency = order.currency
+        default_value = base_order_calculations.base_order_total(order, lines)
+        default_value = TaxedMoney(default_value, default_value)
+        if default_value <= zero_taxed_money(currency):
+            return quantize_price(
+                default_value,
```

Comment:
Is it expected that in case when default_value is < 0, you return it as a total?

Reply:
We have the same condition in checkout total. If we want to refactor it we should do it separate PR.